include/mqt-core/ir/operations/OpType.inc

```diff
@@ -65,11 +65,12 @@ HANDLE_OP_TYPE(36, MultiAFalse, 0, "multi_a_false")
 
 // Neutral atom shuttling operations
 HANDLE_OP_TYPE(37, Move, 0, "move")
-HANDLE_OP_TYPE(38, AodActivate, 0, "aod_activate")
-HANDLE_OP_TYPE(39, AodDeactivate, 0, "aod_deactivate")
-HANDLE_OP_TYPE(40, AodMove, 0, "aod_move")
+HANDLE_OP_TYPE(38, Bridge, 0, "bridge")
+HANDLE_OP_TYPE(39, AodActivate, 0, "aod_activate")
+HANDLE_OP_TYPE(40, AodDeactivate, 0, "aod_deactivate")
+HANDLE_OP_TYPE(42, AodMove, 0, "aod_move")
```

Comment:
```suggestion
HANDLE_OP_TYPE(42, Bridge, 0, "bridge")
HANDLE_OP_TYPE(38, AodActivate, 0, "aod_activate")
HANDLE_OP_TYPE(39, AodDeactivate, 0, "aod_deactivate")
HANDLE_OP_TYPE(40, AodMove, 0, "aod_move")
```
Really minor, but to not break binary compatibility with previous releases, we should not be changing existing enum values.
This should not affect any other functionality in this PR.